src/test/tor/minimal/shadow.data.template/hosts/relay3/fingerprint

```diff
@@ -0,0 +1 @@
+relay3guard 3BAEAC8E24C87B4E536484837B67966487A93214
```

Comment:
I think this should this just be `relay3` since the relay's name is `relay3`?

Reply:
Done